lib/block/blockupload.js

```diff
@@ -128,15 +128,11 @@ class BlockUpload extends EventEmitter {
      */
     async uploadFiles(options = {}) {
         const preferredPartSize = options.preferredPartSize || DEFAULT_PART_SIZE;
-        const maxConcurrent = (options.concurrent && options.maxConcurrent) || DEFAULT_MAX_CONCURRENCY;
-        const retryOptions = {
-            retryMaxCount: options.retryCount || DEFAULT_RETRY_COUNT,
-            timeout: options.timeout,
-            retryAllErrors: options.retryAllErrors,
-            retryMaxDuration: options.retryMaxDuration,
-            retryInitialDelay: options.retryInitialDelay,
-            retryBackoff: options.retryBackoff,
-        };
+        const maxConcurrent = options.maxConcurrent || DEFAULT_MAX_CONCURRENCY;
+
+        if (!options.retryMaxCount) {
+            options.retryMaxCount = DEFAULT_RETRY_COUNT;
```

Comment:
@pheenomenon any reasoning behind using a default `retryMaxCount`?
[this overwrites the value of `retryMaxDuration`](https://github.com/adobe/node-httptransfer/blob/586e992893e21a85f20f7552b63a7fff90bb7c4e/lib/retry.js#L22), which will break behavior in our case since the SDK/pipeline passes a value of 15mins as the `retryMaxDuration`: https://github.com/adobe/asset-compute-pipeline/blob/master/lib/storage/http.js#L66
so this will cause the retry logic to change from 15mins per chunk to 5 retries per chunk

Reply:
@jdelbick no specific reason. it makes sense to refactor this per the SDK/pipeline api requirements.

Reply:
OK, I will remove the default value of 5 retries for now.